plugins/bootc.py

```diff
@@ -0,0 +1,220 @@
+            elif self.opts.json:
+                self.extargs.append("--json")
+            elif self.opts.jsonpath:
+                self.extargs.append("--jsonpath=%s" % self.opts.jsonpath)
+            elif self.opts.booted:
+                self.extargs.append("-b")
+            elif self.opts.pending_exit_77:
+                self.extargs.append("--pending-exit-77")
+            elif self.opts.peer:
+                self.extargs.append("--peer")
+            elif self.opts.installroot:
+                self.extargs.append("--sysroot=%s" % self.opts.installroot)
+        elif cmd == "install":
+            if self.opts.quiet:
+                self.extargs.append("-q")
+            elif self.opts.installroot:
+                self.extargs.append("--sysroot=%s" % self.opts.installroot)
+            elif self.opts.peer:
+                self.extargs.append("--peer")
+            elif self.opts.assumeyes:
+                self.extargs.append("-y")
+            elif self.opts.assumeno:
+                self.extargs.append("-n")
+            elif self.opts.cacheonly:
+                self.extargs.append("-C")
+            elif self.opts.downloadonly:
+                self.extargs.append("--download-only")
+            elif self.opts.releasever:
+                self.extargs.append
+                self.extargs.append("--releasever=%s" % self.opts.releasever)
+            elif len(self.opts.repos_ed) > 0:
+                enabled = set()
+                disabled = set()
+
+                for name, state in self.opts.repos_ed:
+                    if state == "enable":
+                        enabled.add(name)
+                    elif state == "disable":
+                        disabled.add(name)
+
+                if len(list(enabled)) > 0:
+                    for repo in list(enabled):
+                        self.extargs.append("--enablerepo=%s" % repo)
+
+                if len(list(disabled)) > 0:
+                    for repo in list(disabled):
+                        self.extargs.append("--disablerepo=%s" % repo)
+            elif self.opts.uninstall:
+                for pname in self.opts.uninstall:
+                    self.extargs.append("--uninstall=%s" % pname)
+            elif self.opts.apply_live:
+                self.extargs.append("-A")
+            elif self.opts.force_replacefiles:
+                self.extargs.append("--force-replacefiles")
+            elif self.opts.reboot:
+                self.extargs.append("-r")
+            elif self.opts.allow_inactive:
+                self.extargs.append("--allow-inactive")
+            elif self.opts.idempotent:
+                self.extargs.append("--idempotent")
+            elif self.opts.unchanged_exit_77:
+                self.extargs.append("--unchanged-exit-77")
+
+        if self.opts.pkgspec is not None and len(self.opts.pkgspec) > 0:
+            self.extargs += self.opts.pkgspec
+        else:
+            # ensure we have a valid subcommand
```

Comment:
This breaks `bootc status`, this check should only be on commands that expect pkgspecs:

```
# dnf4 bootc status
Missing package specification on bootc sub-command status.
```

Reply:
Whoops.